lgc/state/PalMetadata.cpp

```diff
@@ -383,7 +383,7 @@ void PalMetadata::mergeFromBlob(llvm::StringRef blob, bool isGlueCode) {
       if (destNode->getKind() == msgpack::Type::UInt)
         *destNode = destNode->getUInt() | srcNode.getUInt();
       else if (destNode->getKind() == msgpack::Type::Boolean)
-        *destNode = destNode->getBool() | srcNode.getBool();
+        *destNode = destNode->getBool() || srcNode.getBool();
```

Comment:
This is unrelated, but I don't mind if you want to leave it in the patch.

Reply:
There's the exact same thing two lines above...